web/themes/custom/server_theme/templates/server-theme-person-card.html.twig

```diff
@@ -0,0 +1,48 @@
+<div class="flex flex-col items-start p-0 relative bg-white shadow rounded-lg border-gray-300 relative">
+	{# Leading content #}
+	<div class="flex flex-col items-center p-8 gap-6 w-full flex-none self-stretch grow-0">
+		{# Avatar #}
+		<figure class="flex-none grow-0 flex flex-col justify-center items-center p-0 rounded-full overflow-hidden">
+			{{ image }}
+		</figure>
+		{# Text #}
+		<div class="flex-none grow-0 self-stretch flex flex-col justify-center items-center p-0 g-2">
+			{# Title #}
+			<div class="w-full not-italic font-medium text-sm leading-5 text-center text-gray-900 flex-none self-stretch grow-0">
```

Comment:
Twig looks over verbose. Is the `self-stretch grow-0` for example needed everywhere? Why do we have `flex-none` here? Probably quite a few other classes.

Reply:
Updated